lib/mastodon/emoji_cli.rb

```diff
@@ -33,17 +33,18 @@ def self.exit_on_failure?
       With the --unlisted option, the processed emoji will not be
       visible in the emoji picker (but still usable via other means)
     LONG_DESC
+
     def import(path)
       imported = 0
-      skipped  = 0
-      failed   = 0
+      skipped = 0
```

Comment:
Please don't "fix" variable alignment. You should try to do it to when it makes sense.

Reply:
Sorry, RubyMine did that.